### ISSUE TITLE
Update Divertissements pour la guitare, n°3 Allegretto by SorF

### ISSUE THREAD

**Issue by glenl** [MutopiaProject/MutopiaProject#495]
Mutopia link: [Divertissements pour la guitare, n°3 Allegretto by SorF (at 2.2.2)](http://www.mutopiaproject.org/cgibin/piece-info.cgi?id=453)

Repository: SorF/O1/sor_op1_3

Part of [LilyPond update #4 milestone](https://github.com/MutopiaProject/MutopiaProject/milestones/LilyPond%20update%20%234)

**Comment by fedelibre** [comment]
Can I change the source used?
It seems that the transcription is based on a facsimile from Frederick Noad (which I suppose it's not in the public domain). Can I use the [edition present in IMSLP](http://imslp.org/wiki/6_Divertissements,_Op.1_(Sor,_Fernando))?

**Comment by glenl** [comment]
3, 4, and 6 have been updated to 2.16 without changing the copyright so, for now, let's leave it at that. I would prefer the Simrock edition you reference in IMSLP but because this is a collection it would be good to maintain consistency.

All pieces in the collection should also have the same copyright and I noticed that, for this piece, the copyright on the site differs from that in the LilyPond file. On pieces 3, 4, and 6, the copyright was updated to CC Share Alike 3.0. There is no mention of this in the log so I'll check with Chris in a separate email.
